File: backend/Model/Solicitud.cs
namespace backend.Model;

public class Solicitud
{
    public int Id { get; set; }
    public string Titulo { get; set; } = string.Empty;
    public string Descripcion { get; set; } = string.Empty;
    public string Prioridad { get; set; } = string.Empty;
    public string Estado { get; set; } = string.Empty;

    public Solicitud() { }
}

File: frontend/src/app/solicitud/solicitud.ts
export interface Solicitud {
    id: number;
    titulo: string;
    descripcion: string;
    estado: 'Pendiente' | 'En Proceso' | 'Completada' | 'Cancelada' | string;
    prioridad: 'Baja' | 'Media' | 'Alta' | string;
}

export interface SolicitudCreate {
    titulo: string;
    descripcion: string;
    estado: 'Pendiente';
    prioridad: 'Baja' | 'Media' | 'Alta';
}